[2023-09-06T05:14:37.302+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: filtering_customer_consumption_backup.create_table manual__2023-09-06T05:14:24.102041+00:00 [queued]>
[2023-09-06T05:14:37.337+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: filtering_customer_consumption_backup.create_table manual__2023-09-06T05:14:24.102041+00:00 [queued]>
[2023-09-06T05:14:37.338+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-06T05:14:37.339+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-06T05:14:37.339+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-06T05:14:37.378+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-09-06 05:14:24.102041+00:00
[2023-09-06T05:14:37.389+0000] {standard_task_runner.py:55} INFO - Started process 498 to run task
[2023-09-06T05:14:37.396+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'filtering_customer_consumption_backup', 'create_table', 'manual__2023-09-06T05:14:24.102041+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/filtering_customer_consumption_backup.py', '--cfg-path', '/tmp/tmplwcaz_vx']
[2023-09-06T05:14:37.398+0000] {standard_task_runner.py:83} INFO - Job 9: Subtask create_table
[2023-09-06T05:14:37.548+0000] {task_command.py:376} INFO - Running <TaskInstance: filtering_customer_consumption_backup.create_table manual__2023-09-06T05:14:24.102041+00:00 [running]> on host 60d3bee26398
[2023-09-06T05:14:37.688+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ductv50
AIRFLOW_CTX_DAG_ID=filtering_customer_consumption_backup
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-06T05:14:24.102041+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-06T05:14:24.102041+00:00
[2023-09-06T05:14:37.706+0000] {base.py:71} INFO - Using connection ID 'Postgress_1' for task execution.
[2023-09-06T05:14:37.714+0000] {sql.py:315} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS consumption_alcoholic_20230906 (
            category VARCHAR(255),
            sub_category VARCHAR(255),
            aggregation_date DATE,
            millions_of_dollar INT,            
            pipeline_exc_datetime TIMESTAMP
        );
    
        CREATE TABLE IF NOT EXISTS consumption_cereals_bakery_20230906 (
            category VARCHAR(255),
            sub_category VARCHAR(255),
            aggregation_date DATE,
            millions_of_dollar INT,            
            pipeline_exc_datetime TIMESTAMP
        );
    
        CREATE TABLE IF NOT EXISTS consumption_meats_poultry_20230906 (
            category VARCHAR(255),
            sub_category VARCHAR(255),
            aggregation_date DATE,
            millions_of_dollar INT,            
            pipeline_exc_datetime TIMESTAMP
        );
    , parameters: None
[2023-09-06T05:14:37.768+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=filtering_customer_consumption_backup, task_id=create_table, execution_date=20230906T051424, start_date=20230906T051437, end_date=20230906T051437
[2023-09-06T05:14:37.857+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-06T05:14:38.015+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
